contractsV5/mock/MockHintHandler.sol

```diff
@@ -0,0 +1,113 @@
+pragma solidity 0.5.11;
+
+import "../KyberHintHandler.sol";
+
+
+contract MockHintHandler is KyberHintHandler {
+
+    mapping(address=>bytes8) public reserveAddressToId;
+    mapping(bytes8=>address[]) public reserveIdToAddresses;
+
+    function addReserve(address reserve, bytes8 reserveId) public {
+        reserveIdToAddresses[reserveId].push(reserve);
+        reserveAddressToId[reserve] = reserveId;
+    }
+
+    function parseEthToTokenHint(bytes calldata hint)
```

Comment:
seems these functions should be in hintHandler.
and here only need to implement
IDtoAddress
AddressToId

Reply:
Moved